rtgui/edit.cc

```diff
@@ -747,7 +747,294 @@ void Rectangle::drawToMOChannel(Cairo::RefPtr<Cairo::Context> &cr, unsigned shor
     }
 }
 
-EditSubscriber::EditSubscriber (EditType editType) : ID(EUID_None), editingType(editType), bufferType(BT_SINGLEPLANE_FLOAT), provider(NULL), dragging(false) {}
+void OPIcon::drivenPointToRectangle(rtengine::Coord &pos,
```

Comment:
Sure, done.

Just for my information, is it worth to define all variable as `const` if they are not supposed to be modified ? I don't really know what are the benefit of using `const` in fact, but I guess it is related to optimization and pedantic API ? e.g. for `OPIcon::drawMOImage`, should I switch `img` and `id` to `const` ?